src/dev/build/tasks/bin/scripts/opensearch-dashboards

```diff
@@ -15,8 +26,15 @@ done
 
 DIR="$(dirname "${SCRIPT}")/.."
 CONFIG_DIR=${OSD_PATH_CONF:-"$DIR/config"}
-NODE="${DIR}/node/bin/node"
+
+if [ $OS = "freebsd" ]; then
+  NODE="/usr/local/bin/node"
+else
+  NODE="${DIR}/node/bin/node"
```

Comment:
NODE=`which node`

Reply:
Just want to clarify here - are you planning on making additional changes?

Reply:
Just added the improved and more liberal node selection (OS independent)